EasyLearn/app/helpers/search_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SearchHelper
+end
```

Comment:
If you don't use this file you can delete it.

Reply:
Done.